tests/ci_reset_hw.py

```diff
@@ -0,0 +1,31 @@
+from panda import Panda, PandaDFU
+from libs.resetter import Resetter  # pylint: disable=import-error
```

Comment:
shouldn't need a pylint exception here

Reply:
pre-commit hook and pylint tells that I need it